routes/routes.go

```diff
@@ -55,11 +58,16 @@ func Create(scheduler types.Scheduler) *mux.Router {
 
 	router.PathPrefix("/static/").
 		Handler(
-		http.StripPrefix(
-			"/static/", http.FileServer(
-				&assetfs.AssetFS{Asset: assets.Asset, AssetDir: assets.AssetDir, AssetInfo: assets.AssetInfo, Prefix: "static"})))
+			http.StripPrefix(
+				"/static/", http.FileServer(
+					&assetfs.AssetFS{Asset: assets.Asset, AssetDir: assets.AssetDir, AssetInfo: assets.AssetInfo, Prefix: "static"})))
 
-	router.NotFoundHandler = http.HandlerFunc(notFound)
+	router.NotFoundHandler = authWrap(http.HandlerFunc(notFound))
```

Comment:
We should only be adding the authWrap when authentication is enabled rather than checking in the wrapper. Also lets have a command line flag like `--auth=basic` to enabled it to allow for other authorization schemes in the future.